en/xml/html/topic.chunking.xml

```diff
@@ -0,0 +1,440 @@
+        etc. are written to the same file that contains the <tag>sect1</tag>
+        element.
+      </para>
+      <para> To control the chunking process for sections, use the parameter
+          <parameter>chunk.section.depth</parameter>. By default, the parameter
+        is set to <literal>1</literal> which is equivalent to chunk only level
+        one sections. Setting <parameter>chunk.section.depth</parameter> to
+          <literal>2</literal> has the following effect: </para>
+      <screen># chunk.section.depth=2, chunk.first.sections=1
+chapter         --> ch01.html
+  sect1         --> ch01s01.html
+    sect2       --> ch01s01s01.html
+      sect3
+        sect4
+    sect2       --> ch01s01s02.html
+  sect1         --> ch01s02.html</screen>
+      <para>
+        As you can see, with the value of <literal>2</literal>, level two
+        sections (<tag>sect2</tag>) are written to a separate file.
+        A value of <literal>3</literal> has the following effect:</para>
+      <screen># chunk.section.depth=3, chunk.first.sections=1
+chapter         --> ch01.html
+  sect1         --> ch01s01.html
+    sect2       --> ch01s01s01.html
+      sect3     --> ch01s01s01s01.html
+        sect4
+    sect2       --> ch01s01s02.html
+  sect1         --> ch01s02.html</screen>
+      <para>In other words, parameter <parameter>chunk.section.depth</parameter>
+      cuts at the respective section level.
+      </para>
+    </section>
+    <section xml:id="sec.html.chunk.use.id.as.filename">
+      <title>Create Stable File Names through IDs</title>
+      <para> The previous sections used <emphasis>predictable</emphasis> file
+        names, but <emphasis>not stable</emphasis> ones. If you add or remove a
+        section or chapter, the numbering of the chapters and sections will
+        change and as such the file names too. If you want to share a link, this
+        naming scheme is not useful as it is not stable. </para>
+      <para>Stable file names are not affected when you restructure your
+      document. If you add or remove a structural element, the file names will
+      still be the same.
+      </para>
+      <para>To create such stable file names, use the parameter
+          <parameter>use.id.as.filename</parameter>. This creates a file name
+        through the <tag class="attribute">xml:id</tag> attribute of your
+        component. However, you should keep in mind some issues when you use
+        this naming scheme: </para>
+      <itemizedlist>
+        <listitem>
+          <formalpara>
+            <title>Validate your document before you transform it</title>
+            <para>
+              Validating your document shows you any problems with IDs. For
+              example, double IDs, missing IDs, and syntactically wrong IDs.
+              This is very useful as the DocBook XSL stylesheets do not check
+              for file names which occur twice. This could lead to a situation
```

Comment:
with -> which

Reply:
Ok, I've pushed the above two changes in commit 6448c5d. If you are ok with my changes, can you approve them or comment in case I forgot anything?

Thanks! After you have approved it, I'll merge it to develop. :+1: